src/v2/authentication.rs

```diff
@@ -8,6 +8,8 @@ use serde::{Deserialize, Serialize};
 use time::OffsetDateTime;
 use url::Url;
 
+const MIN_SCOPE: &str = "openid urn:zitadel:iam:org:project:id:zitadel:aud ";
```

Comment:
probably not important at all
```suggestion
const MIN_SCOPE: &str = "openid urn:zitadel:iam:org:project:id:zitadel:aud";
```

Reply:
That space was actually important... We have two scopes in this string. I'm changing how I do so we don't have problems in the future.